misc/ls-extensions/modules/performance-analyzer-services/src/test/java/io/ballerina/PerformanceAnalyzerTest.java

```diff
@@ -38,70 +38,82 @@
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutionException;
 
+import static io.ballerina.PerformanceAnalyzerNodeVisitor.ACTION_INVOCATION_KEY;
+import static io.ballerina.PerformanceAnalyzerNodeVisitor.ENDPOINTS_KEY;
+import static io.ballerina.PerformanceAnalyzerService.MESSAGE;
+import static io.ballerina.PerformanceAnalyzerService.SUCCESS;
+import static io.ballerina.PerformanceAnalyzerService.TYPE;
```

Comment:
Do not consider the visitor as a place to store such keys/ constants. Add a separate class to hold all these keys/ constants

Reply:
Updated